back/src/app/controllers/authenticate.ts

```diff
@@ -45,4 +45,17 @@ export default class authenticateController {
 		req.logout();
 		res.redirect('/');
 	}
+
+	static checkNotAuth(req: Request, res: Response, next: Function) {
+		if (req.user)
+			res.send({type: "error", data: "a user should not be authenticated to acces this path"});
+		next();
+	}
+
+	static checkAuth(req: Request, res: Response, next: Function) {
+		if (!!req.user)
+			res.send({type: "error", data: "a user should not authenticated to acces this path"});
```

Comment:
C'est un détail, le message ici c'est pas "a user should be authenticated to acces this path" plutôt ?

Reply:
oui t'as raison